### ISSUE TITLE
Sketch design templates

### ISSUE THREAD

**Issue by korobochkin** [candy-chat/candy#404]
Maybe you will like it. But anyway thank you for a Chat and this repository :+1:

![home](https://cloud.githubusercontent.com/assets/477498/8712704/3641f7ac-2b68-11e5-9382-0a94a5903516.png)

Another one [at Behance](https://www.behance.net/wip/1261065/2204853)

**Comment by benlangfeld** [comment]
I like the second revision on Behance. What's your plan with this?

**Comment by korobochkin** [comment]
I don't known. Just a sketch. But if someone want to update (refresh) the design of Candy homepage I can try to create a full version of the page. And related to #222 also update the Candy interface design.

**Comment by melissanoelle** [comment]
If you update Candy's interface, you'll want to do it using React and also look at decoupling the interface from it's Core back-end. There are a lot of classes and id's that are required to be present in certain locations for anything to work.

I really like how your second screenshot via the Behance link looks with the added hero space color. It brings up an interesting question though: are Candy's "brand" colors still red and white? If so, the top gradient should probably use them.

If you're not super comfortable building a web page, I can help review your branch/PR if you like. However, if it's time you lack, I can build it out this weekend. Additionally, there's always the "no thank you" route in which I do nothing. ;)

**Comment by tylermauthe** [comment]
Agreed about using React and decoupling... I'm not married to React or anything though. Customising Candy is perilous because of the coupling, so that seems like a thing to try to minimise. The CSS classes are fairly fragile and intertwined too.

IMHO, a new front-end should be heavily influenced by (if not conforming to the spec of) Web Components: (IE: https://github.com/webcomponents/gold-standard/wiki). There are a number of libraries, including the excellent React, that can achieve this.

I am also happy to help out this weekend with building that page, reviewing PRs or just chatting a bit about Reactifying the interface. Basically what @melissanoelle said goes for me too. :+1:

**Comment by melissanoelle** [comment]
I like React because it's got a really well-documented lifecycle and doesn't have a dependency on jQuery, and when I tried a few out, I liked React the best. It's states and props are really nice so long as data is flowing down to children, and only just minorly annoying if you need to flow that data back up to the parent. 

I just want to stress that it's _very_ perilous to customize Candy. It's why a lot of my plugins don't have updates/the correct CSS all the time... they eventually just got so tightly knit into each other that separating it was an entire job in and of itself again, so I stopped updating them. Ain't nobody got time for that. :(

I'd also propose that Candy considers ditching the dependency on jQuery as well.

**Comment by benlangfeld** [comment]
There are several things about Candy's implementation that need changing, such as decoupling UI from core and using something like React/Polymer/whatever; I think the whole community is on-board with that.

Candy's look visually (both itself and the website) feel dated and could certainly use sprucing up, and this thread is an ideal place for people to collaborate on that. Updating the website (gh-pages branch) should be simple and mockups can always be done for a new UI for Candy itself out of the box.

Might I, however, suggest that we treat that topic in a separate thread, and limit this issue to discussing visuals? I would prefer to keep these two subjects as decoupled as possible.

**Comment by tylermauthe** [comment]
@benlangfeld Agreed, sorry! I couldn't help myself from chiming in.

I've created #405 for that discussion.

**Comment by korobochkin** [comment]
My first idea — update the official website. But update the Candy interface also sounds great. I'm not familiar with React or something, I'm just a designer :+1:. @melissanoelle great notice about brand colors. So we need a plan to update all of this. Checkout #405 for the few new interface templates.

**Comment by tylermauthe** [comment]
I think this issue is the place to talk about visuals, I intended #405 to be a place to discuss the decoupling / Reactifying of Candy.
